File: fix.py
import codecs

filepath = 'backend/server.js'

with codecs.open(filepath, 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('let final_pic_id = null;\n      const input_pic_id', 'let final_pic_id = null;\n      let foundPic = null;\n      const input_pic_id')
content = content.replace('const foundPic = picCheck.rows[0];', 'foundPic = picCheck.rows[0];')

with codecs.open(filepath, 'w', encoding='utf-8') as f:
    f.write(content)

print("Fixed")

File: fix.js
const fs = require('fs');
let code = fs.readFileSync('backend/server.js', 'utf8');
code = code.replace('let final_pic_id = null;', 'let final_pic_id = null;\n      let foundPic = null;');
code = code.replace('const foundPic = picCheck.rows[0];', 'foundPic = picCheck.rows[0];');
fs.writeFileSync('backend/server.js', code, 'utf8');
console.log('Fixed!');
